src/CMakeLists.txt

```
@@ -138,3 +138,8 @@ if (NFD_INSTALL)
     FILE ${TARGET_NAME}-config.cmake
   )
 endif()
+
+option(NFD_OVERRIDE_RECENT_WITH_DEFAULT "Use defaultPath instead of recent folder on Windows" ON)
```

Could we set the default to `OFF`?  This will prevent breakage of existing behaviour, and IMO using the recent folder is in the majority of cases the correct thing to do.
```suggestion
option(NFD_OVERRIDE_RECENT_WITH_DEFAULT "Use defaultPath instead of recent folder on Windows" OFF)
```

---

Yeah, was just going to say that I should change the default to OFF, changing that now.